astropy/coordinates/tests/test_iau_fullstack.py

```diff
@@ -29,7 +30,10 @@ def fullstack_icrs():
 def fullstack_fiducial_altaz(fullstack_icrs):
     altazframe = AltAz(location=EarthLocation(lat=0*u.deg, lon=0*u.deg, height=0*u.m),
                        obstime=Time('J2000'))
-    return fullstack_icrs.transform_to(altazframe)
+    with warnings.catch_warnings():  # Ignore remote_data warning
```

Comment:
My fixture-fu is not great, so if there is a better way to do this, please let me know.